dlab_core/schema_validator.py

```diff
@@ -0,0 +1,9 @@
+from jsonschema import validate
```

Comment:
this is abstract class will be better to put in  dlab_core/infrastructure

Reply:
moved